solution/backend/regulations/templates/regulations/partials/sidebar_right.html

```diff
@@ -23,6 +25,7 @@ <h2 id="subpart-resources-heading">
         part="{{ reg_part }}"
         :sections="{{ node_list.sections }}"
         :subparts="{{ node_list.subparts }}"
+
```

Comment:
Super nitpicky but empty line should probably be removed

Reply:
fixed. nitpick away.